### DISCUSSION TITLE
Error on installation: vue, vite, tailwind

### DISCUSSION THREAD

**Discussion by MarkusLei22** [vueform/vueform#97] [Questions] [answered]
Hi, I am having an issue installing vueform. 
I am using:

- "@vueform/vueform": "^1.6.5"
- "vue": "^3.2.41",
- "vite": "^3.1.3",

**Error:**
```
failed to load config from /[...]/apps/frontend/vite.config.ts
node_modules/esbuild/lib/main.js:1336:27: ERROR: [plugin: externalize-deps] No known conditions for "./vite" entry in "@vueform/vueform" package
```

Installation was completed as described in the docs.

- [x] vueform.config.ts
- [x] vite.config.ts
- [x] src/main.ts
- [x] tailwind.config.js

Any help is highly appreciated.
Thank you

**Comment by adamberecz** [comment, marked as answer]
I wasn't able to reproduce this in vite@3. Any chance you can create a reproduction or try a newer version of Vite?

One thing to try that I see people recommending for similar errors: add `"type": "module"` to your `package.json`.
